src/app/telemetry/telemetry.ts

```diff
@@ -0,0 +1,47 @@
+import { ApplicationInsights, SeverityLevel } from '@microsoft/applicationinsights-web';
+
+interface ITelemetry {
+  initialize(): void;
+  trackEvent(eventName: string, payload: any): void;
+  trackException(error: Error, severityLevel: any): void;
+}
+
+class Telemetry implements ITelemetry {
+  private appInsights: ApplicationInsights;
+
+  constructor() {
```

Comment:
Would we ever want to pass in a different config object in the constructor? https://github.com/microsoft/ApplicationInsights-JS#configuration

At a minimum, please review the default configuration settings so that we understand what they are.

Reply:
Reviewed and the defaults are appropriate.